grizli/pipeline/auto_script.py

```diff
@@ -2429,13 +2450,19 @@ def load_GroupFLT(field_root='j142724+334246', PREP_PATH='../Prep', force_ref=No
 
     if files is None:
         files = glob.glob(os.path.join(PREP_PATH, '*fl[tc].fits'))
+        files += glob.glob(os.path.join(PREP_PATH, '*rate.fits'))
         files.sort()
 
     info = utils.get_flt_info(files)
 
     g141 = info['FILTER'] == 'G141'
     g102 = info['FILTER'] == 'G102'
     g800l = info['FILTER'] == 'G800L'
+    gr150cf115w = (info['PUPIL'] == 'F115W') & (info['FILTER'] == 'GR150C')
+    gr150cf150w = (info['PUPIL'] == 'F150W') & (info['FILTER'] == 'GR150C')
+    gr150cf200w = (info['PUPIL'] == 'F200W') & (info['FILTER'] == 'GR150C')
+    gr150rf115w = (info['PUPIL'] == 'F115W') & (info['FILTER'] == 'GR150R')
+    gr150rf150w = (info['PUPIL'] == 'F150W') & (info['FILTER'] == 'GR150R')
 
```

Comment:
Missing the case of `gr150rf200w`

Reply:
Fixed: fef84a382c3aa36b4261c5189d21ac09e9d5261d